lib/atrium/member.rb

```diff
@@ -4,7 +4,10 @@ class Member
     include ::ActiveAttr::Model
 
     attribute :aggregated_at
+    attribute :challenges
```

Comment:
Based on the docs, members don't have `challenges` attribute that will be returned from the API, so I don't think that we need this.

https://atrium.mx.com/docs#members

Reply:
I included the `challenges` attribute because it appears in the response when calling "Read member connection status" if the `connection_status` is `CHALLENGED`. 

It is shown in the 2nd example here: https://atrium.mx.com/docs#read-member-connection-status

Reply:
ahh. I missed that.

Reply:
No worries, thanks @ztoolson :+1: